core/src/Authorization.html

```diff
@@ -265,14 +267,11 @@
       }
     },
     computed: {
-      showUserInfo: ({ userInfo }) => {
-        if (userInfo !== undefined) {
-          if (userInfo.name || userInfo.email) {
-            return true;
-          }
-        }
-      }
-    }
+      showUserInfo: ({ userInfo }) =>
+        Boolean(userInfo && (userInfo.name || userInfo.email)),
+      isUsersPicture: ({ userInfo }) => Boolean(userInfo && userInfo.picture)
```

Comment:
I can't see it being used anywhere... Is it?

Reply:
true